xml/libvirt_configuration_virsh.xml

```diff
@@ -0,0 +1,1231 @@
+    <step>
+     <para>
+      To detach a VF interface, use the <command>virsh
+      detach-device</command> command, which also takes the options listed
+      above.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="libvirt.config.io.pool">
+   <title>Dynamic Allocation of VFs from a Pool</title>
+   <para>
+    If you define the PCI address of a VF into a guest's configuration
+    statically as described in
+    <xref linkend="sec.libvirt.config.io.attach"/>, it is hard to migrate
+    such guest to another host. The host must have identical hardware in the
+    same location on the PCI bus, or the guest configuration must be
+    modified prior to each start.
+   </para>
+   <para>
+    Another approach is to create a <systemitem class="library">libvirt</systemitem> network with a device pool
+    that contains all the VFs of an <xref linkend="vt.io.sriov"/> device.
+    The guest then references this network, and each time it is started, a
+    single VF is dynamically allocated to it. When the guest is stopped, the
+    VF is returned to the pool, available for another guest.
+   </para>
+   <sect3 xml:id="libvirt.config.io.pool.host">
+    <title>Defining Network with Pool of VFs on VM Host Server</title>
+    <para>
+     The following example of network definition creates a pool of all VFs
+     for the <xref linkend="vt.io.sriov"/> device with its physical function
+     (PF) at the network interface eth0 on the host:
+    </para>
+<screen>&lt;network&gt;
+  &lt;name&gt;passthrough&lt;/name&gt;
+    &lt;forward mode='hostdev' managed='yes'&gt;
+      &lt;pf dev='eth0'/&gt;
+    &lt;/forward&gt;
+  &lt;/network&gt;</screen>
+    <para>
+     To use this network on the host, save the above code to a file, for
+     example <filename>/tmp/passthrough.xml</filename>, and execute the
+     following commands. Remember to replace eth0 with the real network
+     interface name of your <xref linkend="vt.io.sriov"/> device's PF:
+    </para>
+<screen><prompt>tux &gt; </prompt><command>sudo virsh net-define /tmp/passthrough.xml</command>
+<prompt>tux &gt; </prompt><command>sudo</command> virsh net-autostart passthrough
+<prompt>tux &gt; </prompt><command>sudo</command> virsh net-start passthrough</screen>
+   </sect3>
+   <sect3 xml:id="libvirt.config.io.pool.guest">
+    <title>Configuring VM Guest to Use VF from the Pool</title>
+    <para>
+     The following example of guest device interface definition uses a VF of
+     the <xref linkend="vt.io.sriov"/> device from the pool created in
+     <xref linkend="libvirt.config.io.pool.host"/>. <systemitem class="library">libvirt</systemitem> automatically
+     derives the list of all VFs associated with that PF the first time the
+     guest is started.
+    </para>
+<screen>&lt;interface type='network'&gt;
+  &lt;source network='passthrough'&gt;
+&lt;/interface&gt;</screen>
+    <para>
+     To verify the list of associated VFs, run <command>virsh net-dumpxml
```

Comment:
'To verify the list of associated VFs, run [...] on the host after the first guest that uses the network with the pool of VFs starts.' is hard to read. Maybe rephrase to:
 'After the first guest starts that uses the network with the pool of VFs, verify the list [...]. Do so by running [...] on the host:'